test/functional/rpc_dumptxoutset.py

```
@@ -49,9 +49,13 @@ def run_test(self):
             out['txoutset_hash'], 'b2d7429106c96f5ab831843d5c96ba131ca8793111d0a0e30e7d7d8b4841e6cc')
         assert_equal(out['nchaintx'], 101)
 
-        # Specifying a path to an existing file will fail.
+        # Specifying a path to an existing or invalid file will fail.
         assert_raises_rpc_error(
             -8, '{} already exists'.format(FILENAME),  node.dumptxoutset, FILENAME)
```

Ruff (0.12.2)

Use f-string instead of `format` call

Convert to f-string

(UP032)